src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDeviceTimeCheck/DMDeviceTimeCheckViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+#if !RELEASE
+
+import UIKit
+
+final class DMDeviceTimeCheckViewController: UITableViewController {
+
+	// MARK: - Init
+
+	init(store: Store) {
+		self.viewModel = DMDeviceTimeCheckViewModel(store: store)
+		super.init(style: .insetGrouped)
+	}
+
+	required init?(coder: NSCoder) {
+		fatalError("not supported")
+	}
+
+	// MARK: - Overrides
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setupTableView()
+		title = "Device Time Check 📱 🕰"
+	}
+
+	// MARK: - Protocol UITableViewDataSource
+
+	override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		return viewModel.itemsCount
+	}
+
+	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		guard let cell = tableView.dequeueReusableCell(withIdentifier: "DMSwitchTableViewCell") as? DMSwitchTableViewCell else {
+			let dummy = UITableViewCell(style: .default, reuseIdentifier: "DummyFallBackCell")
+			dummy.textLabel?.text = "Fallback cell"
+			return dummy
+		}
+		cell.configure(cellViewModel: viewModel.cellViewModel(for: indexPath))
+		return cell
+	}
+
+	// MARK: - Public
```

Comment:
Shouldn't we remove the unused ones?

Reply:
Hm feels like we don't have no clear decision yet. I personally like to keep them in for future me, if I have to add something in an currently empty Mark